### ISSUE TITLE
FORS change of identifiers

### ISSUE THREAD

**Issue by john-shepherdson** [cessda/cessda.cdc.versions#640]
From: Guillaume Lefebvre 
Date: Fri, 8 Mar 2024 at 10:59
Subject: Question about OAI-PMH harvesting and identifier change
To: John Shepherdson

As you know, our application SWISSUbase is currently harvested by the CESSDA catalogue.

In a near future, we plan to change some bits about how our internal dataset versioning works within SWISSUBase. This would imply that we would change some metadata identifiers currently provided through our OAI-PMH endpoint

First, we will need to change the OAI-PMH record header "identifier" field from something like "oai:swissubase.ch:1000-1-0", to "oai:swissubase.ch:9517409b-6854-48cf-913a-2e84ea4782bd"
Second, we currently provide 2 IDNo within DDI profile, one being a DOI identifier and the other being an internal SWISSUbase identifier. We will drop the latter, only keeping the DOI identifier as unique IDNo.
Basically we we would like to know how to proceed with this ? I expect that records cannot be really updated, since the record "identifier" will change. So:

Shall we ust release our new SWISSUbase version with new versioning scheme, then it will be harvested by CESSDA, duplicates will be created, and CESSDA can then drop the old records ?
Shall we sync our release with CESSDA, so that old records are dropped first, and then new ones are harvested ?
Do you have another solution to propose ?

**Comment by john-shepherdson** [comment]
I think we should drop their current records then harvest the news ones. Any reason why not?

**Comment by matthew-morris-cessda** [comment]
I don't think we need to do anything. The old records will be automatically deleted and having 2 `IDNo` elements isn't an issue, as we deal with this in other repositories.

**Comment by matthew-morris-cessda** [comment]
@MortenSikt Could you follow this up with SWISSUbase?

**Comment by MortenSikt** [comment]
From Guillaume: 

> Hello Morten,
> Thanks for contacting me. The internal identifiers of FORS datasets were indeed changed last year when we released the SWISSUbase version 2.8.0 on the 18th of June 2024. We did this based on the following answer I got from John:
> Guillaume,
> 
> Just go ahead and make the changes as described. We expect the new records to supersede the old ones, with no need for any action on our part.
> 
> Please notify me when the changes have been made, so that I can take a look at CDC to make sure everything is correct.
> 
> Regards,
> 
> John
> John Shepherdson BSc (Hons) MSc CEng MIET
> CESSDA Platform Delivery Director
> 
> 
> We checked the CDC a few weeks after the release, and everything seemed indeed to have gone well.
> Best regards,
> Guillaume

So everything has worked fine then, I am closing

**Comment by matthew-morris-cessda** [comment]
Thanks Morten
